typescript/thing-description/schema/td-json-schema-validation.json

```diff
@@ -1,6 +1,6 @@
 {
   "title": "Thing Description",
-  "version": "1.1-05-September-2022",
+  "version": "1.1-23-March-2023",
```

Comment:
Since I stumbled over this change. I don't think this change should be in this PR.

Reply:
It's been pushed by a bot...

Reply:
Let revert the change, decided in Call 2023-04-17